adb-expresslane/list-workshops/list-workshops.md

```diff
@@ -15,7 +15,7 @@ Always select, **Run on Your Tenancy** when starting a workshop.
 
 ## Autonomous Database 1: Autonomous Database 15 Minute Quick Start
 
-In just 15 minutes, deploy an Autonomous Database optimized for data warehousing workloads, integrate that warehouse with a data lake, and then gain meaningful business insights.
+In just 15 minutes, deploy an Autonomous Database optimized for data warehousing workloads, integrate that warehouse with a data lake, and then gain meaninserverlessgful business insights.
```

Comment:
@martygubar - could you fix this typo meaninserverlessgful in the list-workshops.md file